src/composables/useTooltips.ts

```diff
@@ -0,0 +1,75 @@
+import { Layer } from "leaflet";
+import { LGeoJson } from "@vue-leaflet/vue-leaflet";
+import { Feature } from "geojson";
+import { Dict } from "../types/utilTypes";
+import { FeatureIndicatorValues, AppConfig, FeatureWithColour } from "../types/resourceTypes";
+import { getFeatureId } from "../resources/utils";
+import { APP_BASE_ROUTE } from "../router/utils";
+
+const router = useRouter();
+
+export const useTooltips = (
+    appConfig: Ref<AppConfig>,
+    featureRefs: Ref<(typeof LGeoJson)[]>,
+    selectedIndicator: Ref<string>,
+    selectedIndicators: Ref<Dict<FeatureIndicatorValues>>,
+    selectedCountryId: Ref<string>
+) => {
+    const tooltipForFeature = (feature: Feature) => {
+        let indicatorValues = "";
+        const featureId = getFeatureId(feature, appConfig.value);
+        if (featureId in selectedIndicators.value) {
+            const featureValues = selectedIndicators.value[featureId];
+            indicatorValues = Object.keys(featureValues)
+                .sort((key) => (key === selectedIndicator.value ? -1 : 1))
```

Comment:
Rather than doing this sort, and human readable name lookup, for every feature, let's make a computed which sorts the indicators and maps them to human readable name - can then loop through those for each feature.